website/content/en/preview/troubleshooting.md

```diff
@@ -433,6 +433,14 @@ To correct the problem if it occurs, you can use the approach that AWS EBS uses,
 ]
 ```
 
+### Node is not deleted, even though `ttlSecondsUntilExpired` is set or the node is empty
```

Comment:
Should this be in the deprovisioning section of the troubleshooting docs?

Reply:
Maybe? I put it there since it's caused by a Node Launch issue, but maybe it's easier to find under deprovisioning.  I can move it.

Reply:
Actually, that just links back to node not initialized.  Let's leave it as is for now until someone has time to rework all this. I just want to get the log messages on the site so they are searchable.